react-basic-practice/src/components/TableColumn/index.tsx

```diff
@@ -71,13 +81,19 @@ const TableColumn: ICustomerColumn[] = [
   },
   {
     key: "options",
-    value: (data: ICustomer) => (
-      <OptionMenu
-        onView={() => console.log(`View ${data.id}`)}
-        onEdit={() => console.log(`Edit ${data.id}`)}
-        onDelete={() => console.log(`Delete ${data.id}`)}
-      />
-    ),
+    value: (
+      data: ICustomer,
+      action?: (type: ActionType, id?: number) => void
+    ) => {
+      const handlers = createOptionMenuHandler(action!, data);
```

Comment:
why we use `!`? Pls check condition clearer

Reply:
i done